### PR TITLE
Update why-wasnt-my-application-for-a-student-developer-pack-approved.md

### PR DESCRIPTION
**Pull request by VibhuRajput**
### Why:
Made the document less ambiguous 

### What's being changed:
The initial doc was good but I made it more easy to understand for beginners and reduced the ambiguity by using descriptive language.
READY FOR REVIEW 

### Check off the following:
- [x] I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
- [x] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [x] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

**Comment by janiceilene** [comment]
Thanks for opening a PR 👍 Please fill out the [pull request template](https://raw.githubusercontent.com/github/docs/main/.github/PULL_REQUEST_TEMPLATE.md) in the top comment in this pull request, then we'll get this up for review!

**Comment by VibhuRajput** [comment]
The initial doc was good but I made it more easy to understand for beginners and reduced the ambiguity by using descriptive language.

File: content/education/explore-the-benefits-of-teaching-and-learning-with-github-education/why-wasnt-my-application-for-a-student-developer-pack-approved.md [hunk 1]
```diff
@@ -19,7 +19,9 @@ versions:
 
 ### Unclear academic affiliation documents
 
-If the image you uploaded doesn't clearly identify your current academic status, we require further proof of your academic status. {% data reusables.education.upload-proof-reapply %}
+If the dates or schedule mentioned in your uploaded image doesn't matches our eligibility criteria, we require further proof of your academic status.
```
Comment (HonkingGoose) [inline content/education/explore-the-benefits-of-teaching-and-learning-with-github-education/why-wasnt-my-application-for-a-student-developer-pack-approved.md:22]:
```suggestion
If the dates or schedule mentioned in your uploaded image do not match our eligibility criteria, we require further proof of your academic status.
```

File: content/education/explore-the-benefits-of-teaching-and-learning-with-github-education/why-wasnt-my-application-for-a-student-developer-pack-approved.md [hunk 2]
```diff
@@ -58,6 +60,7 @@ For more information, see:
 
 You're ineligible for a {% data variables.product.prodname_student_pack %} if:
 - You're enrolled in an informal learning program that is not part of the [{% data variables.product.prodname_dotcom %} Campus Program](https://education.github.com/schools) and not enrolled in a degree or diploma granting course of study.
+You're studying in a degree which is about to get terminated in current academic session (such as for 2021. If you're about to pass your degree by the mid of 2021.
```
Comment (HonkingGoose) [inline content/education/explore-the-benefits-of-teaching-and-learning-with-github-education/why-wasnt-my-application-for-a-student-developer-pack-approved.md:63]:
```suggestion
You're pursuing a degree which will be terminated in the current academic session.
```

**Review by HonkingGoose — COMMENTED** [review]
What do you think of this wording?

**Comment by janiceilene** [comment]
@VibhuRajput Thanks so much for filling out the template! I've triaged this for the team ✨

**Comment by VibhuRajput** [comment]
> @VibhuRajput Thanks so much for filling out the template! I've triaged this for the team ✨

Thanks @janiceilene

**Review by MMLaws7 — CHANGES_REQUESTED** [review]
I’m just asking for my changes

**Review by MMLaws7 — APPROVED** [review]

File: content/education/explore-the-benefits-of-teaching-and-learning-with-github-education/why-wasnt-my-application-for-a-student-developer-pack-approved.md [hunk 3]
```diff
@@ -58,6 +60,7 @@ For more information, see:
 
 You're ineligible for a {% data variables.product.prodname_student_pack %} if:
 - You're enrolled in an informal learning program that is not part of the [{% data variables.product.prodname_dotcom %} Campus Program](https://education.github.com/schools) and not enrolled in a degree or diploma granting course of study.
+You're pursuing a degree which will be terminated in the current academic session.
```
Comment (myarb) [inline content/education/explore-the-benefits-of-teaching-and-learning-with-github-education/why-wasnt-my-application-for-a-student-developer-pack-approved.md:63]:
```suggestion
- You're pursuing a degree which will be terminated in the current academic session.
```

**Comment by myarb** [comment]
Thanks so much for this PR, @VibhuRajput, and that great review, @HonkingGoose 🎉 I've committed your suggestions and we'll get this merged 🔜  ✨

**Review by myarb — APPROVED** [review]
Thanks so much for this PR, @VibhuRajput, and that great review, @HonkingGoose 🎉 I've committed your suggestions and we'll get this merged 🔜  ✨

**Comment by HonkingGoose** [comment]
Cheers @myarb! Thanks for including my suggestions! 🎉

**Comment by R-BLIP885** [comment]
> ### Why:
> Made the document less ambiguous
> 
> ### What's being changed:
> The initial doc was good but I made it more easy to understand for beginners and reduced the ambiguity by using descriptive language.
> READY FOR REVIEW
> 
> ### Check off the following:
> * [x]  I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
> * [x]  For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
> * [x]  For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

**Review by SinEstresOrrantia — APPROVED** [review]

**Pull request merged**